src/create_string.cpp

```diff
@@ -3,12 +3,18 @@
 #include <string.h>
 #include "create_string.h"
 
-char* create_string(Local<Value> value) {
-  if (value->IsNull() || !value->IsString()) {
+char* create_string(Nan::MaybeLocal<v8::Value> maybevalue) {
+  v8::Local<v8::Value> value;
+  
+  if (maybevalue.ToLocal(&value)) {
+    if (value->IsNull() || !value->IsString()) {
+      return 0;
+    }
+  } else {
     return 0;
```

Comment:
> > -char* create_string(Local<Value> value) {
> > -  if (value->IsNull() || !value->IsString()) {
> > +char* create_string(Nan::MaybeLocal<v8::Value> maybevalue) {
> > +  v8::Local<v8::Value> value;
> > +  
> > +  if (maybevalue.ToLocal(&value)) {
> > +    if (value->IsNull() || !value->IsString()) {
> > +      return 0;
> > +    }
> > +  } else {
> >      return 0;
> 
> Could use the `Maybe` API instead of dangerous null pointers. This
> would be `return Nan::Nothing();`, the other case `return
> Nan::Just(str);`

Unfortunately we are leaving a nice V8 world here:

create_string() pointers are being directly fed into C structures.
In fact this function exists only because the libsass C API insists
that almost all (char *) pointers should be heap allocated, because it
says it frees them (I think this API is incorrect, but that's another story).

But ~Utf8String() may crash if someone free()'s that underlying string, am I correct?
And if the string fits n str_st_ it can't be free()d.